### DISCUSSION TITLE
Incorrect date format in Simplified Chinese

### DISCUSSION THREAD

**Discussion by ykla** [community/community#128715] [Mobile]
### Select Topic Area

Bug

### Body

A review of the GitHub mobile app revealed that the majority of translations of dates from Simplified Chinese are erroneous. Japanese translations also exhibited similar issues. 

In particular, GitHub displays the time difference from the present rather than the actual date when referring to dates. For instance, it shows "months ago" or "days ago." In some contexts, the term "ago" can be inferred from the text and thus omitted. However, when displayed as an isolated unit, this is incorrect. Without the term "前," The term "ago" can be mistaken for an actual date. For instance, the term "6 months ago" is translated as "6 月" in Simplified Chinese, which is incorrect (6月 means it occurred at June, 2024, not 6 months ago), leading users to believe the submission occurred in June 2024, when it actually happened around January. The correct translation should be "6 月前" (6 months ago). All appropriate translations should be "xx 日前" (xx days ago), "XX 月前" (xx months ago), "XX 分钟前" (xx minutes ago), "刚才" (just now), and so forth. 

![2{F_O U_FY5DQ66)~NWOHX](https://github.com/community/community/assets/10327999/9d7b722d-b5ff-4dec-a668-c2dd10ddfd93)

![CXB }3@1@$R~I%CRA0D))W4](https://github.com/community/community/assets/10327999/bdcb432a-7a44-47ca-a10c-a4db679719a4)

![Q4 Z6P1DGFP0RY8YBGV% 2O](https://github.com/community/community/assets/10327999/ce45e74e-691b-4db4-be51-5e1c9b705ee2)

![T`6X55I11M4B~TFY7I}V%)N](https://github.com/community/community/assets/10327999/7b19673f-48c3-47d2-bf68-a74eaae3f0fe)

There is only one instance where the date translation is correct. 

Additionally, the translation in the release section is not idiomatic. For example, "xx released this June 13" is currently translated as "xx 发布于 6月13日". The appropriate and idiomatic translation is "xx 发布于 6月13日." The phrase "发布于" should be used in place of "发布于此6月13日." For "xx 已将提交 yy 合并到 zz 3天前," a more idiomatic expression would be "xx 于3天前将 yy 合并到 zz."

![QF@D{%M9U)}RKYW4@MNURYQ](https://github.com/community/community/assets/10327999/66a72d02-0440-4cf0-9920-fa6a6ef13ac9)

It is recommended that consideration be given to the separation of i18n if the project as a whole cannot be open sourced.

**Comment by eliperkins** [comment]
Hey @ykla, thanks for this feedback! As a non-Chinese speaker, getting your input on addressing this is quite helpful. I've filed an issue within our repo to get this resolved by updating the codebases themselves.

Thank you so much for helping keep the quality of these translations high 💖 

> It is recommended that consideration be given to the separation of i18n if the project as a whole cannot be open sourced.

I would love for this to be possible, as well, given beneficial feedback from the community like yourself! Unfortunately, this is not possible at this time due to technical restraints.

However, feedback like yours continues to be critical. Please continue to provide this feedback here in the community and we'll work to get this addressed as quickly as possible!

**Reply by ykla** [reply]
This issue still here,  please keep working.
